docs/source/markdown/podman-systemd.unit.5.md

```diff
@@ -835,6 +836,7 @@ Valid options for `[Kube]` are listed below:
 | AutoUpdate=registry                 | --annotation "io.containers.autoupdate=registry"                 |
 | ConfigMap=/tmp/config.map           | --config-map /tmp/config.map                                     |
 | ContainersConfModule=/etc/nvd\.conf | --module=/etc/nvd\.conf                                          |
+| ExitCodePropagation=how             | How to propagate container error status              |
```

Comment:
Please align `|` to the end of the table

Reply:
ARGH! Thank you for catching that. Fixed, but I will not push until I see how CI does.